File: bubbleSort/bubbleSort.js
let list = [5, 2, 7, 4, 1, 6, 3, 0];
function swapValues(fwd, flw) {
  [list[fwd], list[flw]] = [list[flw], list[fwd]];
}

for (i = 1; i < list.length; i++) {
  //console.log(list[i]);
  for (j = 0; j < list.length - 1; j++) {
    //console.log(i, j);
    //console.log(list);
    if (list[i] < list[j]) {
      /*console.log(
        `swapping index [i] ${i} = ${list[i]} with index [j] ${j} = ${list[j]}`
      );*/
      swapValues(i, j);
      //console.log(list);
    }
  }
}

console.log(list);

File: bubbleSort/bubble_sort.py
list = [5, 2, 7, 4, 1, 6, 3, 0]


def swap_values(fwd, flw):
    [list[fwd], list[flw]] = [list[flw], list[fwd]]


for i in range(1, len(list)):
    for j in range(0, len(list)-1):
        if list[i] < list[j]:
            swap_values(i, j)

print(list)
